### ISSUE TITLE
Extended Table/Column Attribute

### ISSUE THREAD

**Issue by ehsaanwelcome** [Codeer-Software/LambdicSql#2]
Hi,
Table/Column attributes does not work, if, these are extended and then applied. for example,
```
class MyTableAttribute: TableAttribute{
    string extra;
    public Table1(string name, string schema, string extra):base(name. schema){
        this.extra = extra;
    }
}
```

I checked the code where it does
`type.GetCustomAttributes(true).Where(e => e.GetType().FullName == "System.ComponentModel.DataAnnotations.Schema.TableAttribute").FirstOrDefault()`

this one will work for both base and extendedattributed 
```type.GetCustomAttribute(typeof(TableAttribute))```
or
```type.Attribute<TableAttribute>()```

see, if you can integrate this one

**Comment by Ishikawa-Tatsuya** [comment]
Thank you!
This is a bug.
I think you will heal up with the code of your suggestion.
I will fix this soon.

**Comment by Ishikawa-Tatsuya** [comment]
Hi,
This bug is fixed at LambdicSql_β 0.37.0.
We thank you for your issue.
